app/assets/stylesheets/body.scss

```diff
@@ -0,0 +1,21 @@
+.body {
+  background-color: floralwhite;
```

Comment:
already set it in the html

Reply:
I added a variable now and replaced "floralwhite" to $primary-color. 
Without declaring that specifically here the body background is white again (and not floralwhite like the html)
Is there a way to make the color the same without declare it again here?..